setup/ProjectSystemSetup/UnifiedSettings/ManagedProjectSystem.registration.json

```diff
@@ -0,0 +1,111 @@
+{
+    "properties": {
+      "projectsAndSolutions.sdkStyleProjects.fastUpToDateCheck.enabled": {
+        "type": "boolean",
+        "title": "@Setting_FastUpToDateCheck_Enabled_Title;{860A27C0-B665-47F3-BC12-637E16A1050A}",
+        "default": true,
+        "requiresRestart": false,
+        "migration": {
+          "pass": {
+            "input": {
+              "store": "SettingsManager",
+              "path": "ManagedProjectSystem\\FastUpToDateCheckEnabled"
+            }
+          }
+        }
```

Comment:
"Migration" implies that the unified settings will _move_ the current value of the setting as specified in the `path` and store it somewhere else going forward. Is that accurate? Any concerns about round-tripping the settings between VS installs using unified settings and those that don't? Or am I misunderstanding what this means?

Reply:
No concerns about round-tripping, as this migration field takes care of converting legacy <-> unified settings location.
That's correct, they will be stored in a new location determined by the unified settings manager

Reply:
Are settings stored in both locations then, at least for now? I wonder how conflicts are resolved.